modules/ROOT/pages/footnotes.adoc

```diff
@@ -0,0 +1,135 @@
+
+Superscripted footnote numbers can also be dragged to a different position in the document. Footnote numbers and their relationship with the respective footnote entry are automatically updated when they are dragged to a different position.
+
+Each superscripted footnote number links to its respective footnote entry, and each footnote entry is prefixed with a caret — *^* — that links back to its respective superscripted footnote number. This allows for navigation between the editor content and the footnotes section.
+
+If the TinyMCE document is large enough to produce a scroll bar, clicking the caret prefixing a footnote entry scrolls the document back to the line containing said entry’s superscripted footnote number.
+
+NOTE: Although the viewport scrolls back to the line containing the footnote number, the insertion point is not changed. Users must move the insertion point by, for example, clicking the I-beam pointer somewhere in the now-scrolled document.
+
+
+=== CSS
+
+//// explanatory text here.
+
+[source,css]
+----
+include::example$css-codeblock/footnotes-plugin.css[]
+----
+
+
+== Usage notes
+
+=== The update footnotes button
+
+The update footnotes button, as its name implies, updates the state of all footnotes in a {productname} document.
+
+To add the update footnote button to the {productname} toolbar, add `footnotesupdate` to the `toolbar`.
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'footnotes',
+  toolbar: 'footnotes footnotesupdate',
+});
+----
+
+In the main, footnotes update automatically as they are added, removed, or edited. However, there are some circumstances where user-edits change footnote state but the footnotes themselves do not update.
+
+For example, when an image caption with an attached footnote is removed (by unchecking the _Show caption_ checkbox in the *Insert/Edit Image* dialog), the associated footnote text is not automatically removed.
+
+In similar fashion, if a table row or column that contains a footnote is removed, the associated footnote text is not automatically removed.
+
+These orphan footnote texts can be deleted manually. They can also be removed by clicking the `footnotesupdate` button in the toolbar.
+
+
+=== Footnotes and {productname} Previews
+
+*File > Preview* presents footnote-specific in-document links, but these links do not work in the Preview.
+
+
+=== Footnotes and exporting {productname} documents to PDF
+
+In-document footnote links are not preserved when a {productname} document is exported to a PDF file.
+
+The footnotes and footnote text are preserved, but the clickable links between them are not. In an exported PDF file, the superscripted footnote numbers are not clickable.
+
+And the carets set between a footnote number and footnote text, which act as links back to a superscripted footnote number in a {productname} document, are removed in the exported PDF file.
```

Comment:
This doesn't look correct to me... it's an issue with the export plugin, **not** footnote. If anything it should be mentioned there because for example if you export as a PDF via print it works fine (granted the styling is lost which does seem like a bug but the link works).

- [ ] Reconsider documenting this for footnotes, as it does not appear to be a footnotes problem and appears to be elsewhere

Reply:
This is now a ticket: [DOC-1810](https://ephocks.atlassian.net/browse/DOC-1810).